### PR TITLE
Added Github Actions based CI

### PR DESCRIPTION
**Pull request by Steveb-p**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | N/A
| **Target version** | `7.5`
| **BC breaks**      | no

**TODO**:
- [x] ~Implement feature / fix a bug.~
- [x] ~Implement tests.~
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

**Comment by Steveb-p** [comment]
@adamwojs @alongosz I'm unsure, but this test failure looks like an expected behavior. Care to take a look?

**Comment by mnocon** [comment]
Q: do some of the test suites from Github Actions and Travis overlap? If yes: can we remove them from Travis? I can help with that if needed.

**Comment by Steveb-p** [comment]
> Q: do some of the test suites from Github Actions and Travis overlap? If yes: can we remove them from Travis? I can help with that if needed.

@mnocon Currently tests than can be removed from Travis are:
 * Code style checks
 * Unit tests
 * Integration tests without "normal" database (i.e. Sqlite)
 * PostgreSQL integration tests

MySQL, Redis & Solr tests are currently not covered by GH actions.

_EDIT: I can remove them myself, but first I needed confirmation that their GH action variant work and isn't noticeably slower._

**Comment by Steveb-p** [comment]
@mnocon Travis configuration removal for parts covered by this PR can be found in this commit: 2ad555e9.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,113 @@
+name: CI
+
+on:
+    push:
+        branches:
+            - master
```
Comment (adamwojs) [inline .github/workflows/ci.yaml]: There is no `master` branch in this repository.

**Review by adamwojs — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Steveb-p) [reply .github/workflows/ci.yaml]: Fixed in 814080e6.

**Review by alongosz — CHANGES_REQUESTED** [review]
+1 but

Because this PR is not just about moving part of CI from Travis to GH Actions, but also fixes a bug, it needs to be rebased and split into 2 commits:
* Fixed number formatting in Twig FileSizeExtension
* Moved parts of CI from Travis to GitHub Actions

and marked as [FAST FORWARD] + label.

Then I can merge it in a fast-forward way

File: eZ/Publish/Core/MVC/Symfony/Templating/Twig/Extension/FileSizeExtension.php [hunk 2]
```diff
@@ -100,7 +100,9 @@ public function sizeFilter($number, $precision)
         }
         $formatter = new NumberFormatter($this->getLocale(), NumberFormatter::PATTERN_DECIMAL);
         $formatter->setPattern($formatter->getPattern() . ' ' . $this->translator->trans($this->suffixes[$i]));
-        $return = $formatter->format(round($number, $precision));
+        $formatter->setAttribute(NumberFormatter::MAX_FRACTION_DIGITS, $precision);
+
+        $return = $formatter->format($number);
 
         return $return;
```
Comment (alongosz) [inline eZ/Publish/Core/MVC/Symfony/Templating/Twig/Extension/FileSizeExtension.php]: Just noticed it should be:
```suggestion
        return $formatter->format($number);
```

**Review by alongosz — CHANGES_REQUESTED** [review]

File: eZ/Publish/Core/MVC/Symfony/Templating/Twig/Extension/FileSizeExtension.php (hunk 2 above)
Comment (Steveb-p) [reply eZ/Publish/Core/MVC/Symfony/Templating/Twig/Extension/FileSizeExtension.php]: Fixed during rebase.

**Review by alongosz — APPROVED** [review]

**Pull request merged**